selftests/unit/test_loader.py

```diff
@@ -235,22 +235,17 @@ def test_load_simple_not_exec(self):
                                              'avocado_loader_unittest',
                                              mode=DEFAULT_NON_EXEC_MODE)
         simple_test.save()
-        test_class, test_parameters = (
-            self.loader.discover(simple_test.path, loader.ALL)[0])
-        self.assertTrue(test_class == test.NotATest, test_class)
-        test_parameters['name'] = test.TestName(0, test_parameters['name'])
-        test_parameters['base_logdir'] = self.tmpdir
-        tc = test_class(**test_parameters)
-        self.assertRaises(exceptions.NotATestError, tc.test)
+        test_class, _ = self.loader.discover(simple_test.path, loader.ALL)[0]
+        self.assertTrue(test_class == loader.NotATest, test_class)
         simple_test.remove()
 
     def test_load_pass(self):
         avocado_pass_test = script.TemporaryScript('passtest.py',
                                                    AVOCADO_TEST_OK,
                                                    'avocado_loader_unittest')
         avocado_pass_test.save()
-        test_class, test_parameters = (
-            self.loader.discover(avocado_pass_test.path, loader.ALL)[0])
+        test_class, _ = self.loader.discover(avocado_pass_test.path,
```

Comment:
Unrelated to this commit description, but indeed it is a nice cleanup and goes well along the previous change. +1 here